02_snake_animation/main.go

```diff
@@ -29,7 +29,9 @@ type snake struct {
 
 // newSnake returns a new struct instance representing a snake.
 func newSnake() snake {
-	return snake{coord{5, 5}, coord{1, 0}}
+	for {
+		return snake{coord{5, 8}, coord{1, 0}}
+	}
```

Comment:
Why do you need for-loop here? Inside the loop you are just calling a return, so the loop always runs only one iteration?
Please explain or fix.